README.md

```diff
@@ -3,62 +3,93 @@
 [![Build Status](https://api.cirrus-ci.com/github/cirruslabs/cirrus-cli.svg?branch=master)](https://cirrus-ci.com/github/cirruslabs/cirrus-cli)
 
 Cirrus CLI is a tool for running containerized tasks reproducibly in any environment. Run your tasks locally the same way
-they are executed in CI or on your colleague's computer. Immutability of Docker containers ensures your project will compile
-years form now regarding what versions of packages you have locally.
+they are executed in CI or on your colleague's machine. Immutability of Docker containers ensures your project will compile
+years from now regardless what versions of packages you'll have locally.
 
 ![Cirrus CLI Demo](images/cirrus-cli-demo.gif)
 
-**Note:** Even though Cirrus CLI is using the same configuration format as [Cirrus CI](https://cirrus-ci.org/), 
-Cirrus CLI can be used in any environment that has Docker installed including any CI system like Jenkins, Github Actions, etc.
-Think of Cirrus CLI as an executor of Cirrus tasks on a single machine and Cirrus CI as a cloud executor.
-
 ## Installation
 
-### Releases
-
-Check the [releases page](https://github.com/cirruslabs/cirrus-cli/releases) for a pre-built `cirrus` binary for your platform.
-
-### Go
-
-If you have [Go](https://golang.org/) installed, you can run:
-
-```
-go get github.com/cirruslabs/cirrus-cli/...
-```
-
-This will build and place the `cirrus` binary in `$GOPATH/bin`.
-
-To be able to run `cirrus` command from anywhere, make sure the `$GOPATH/bin` directory is added to your `PATH`
-environment variable (see [article in the Go wiki](https://github.com/golang/go/wiki/SettingGOPATH) for more details).
+* [Prebuilt Binary](intallation.md#prebuilt-binary)
+* [Golang](intallation.md#golang)
+* [Github Actions](intallation.md#github-actions)
```

Comment:
When looking at this list rendered in Markdown, it feels like these four CI-related options should be grouped together to separate them from the rest of the "standard" options at first glance.

What do you think?